avm/ptn/subscription/service-health-alerts/tests/e2e/existing-action-group/main.test.bicep

```
@@ -0,0 +1,73 @@
+targetScope = 'subscription'
+
+metadata name = 'Deploying multiple service health alerts with an existing action group.'
+metadata description = 'This instance deploys the module with a reference to an existing action group.'
+
+// ========== //
+// Parameters //
+// ========== //
+
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-ash-${namePrefix}-${serviceShort}-rg'
+
+@description('Required. The subscription ID to deploy service health alerts to. If not provided, the current subscription will be used.')
+param subscriptionId string = subscription().subscriptionId
+
+@description('Optional. The location to deploy resources to.')
+param resourceLocation string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'ashact'
+
+@description('Optional. A token to inject into the name of each resource. This value can be automatically injected by the CI.')
+param namePrefix string = '#_namePrefix_#'
+
+// ============ //
+// nestedDependencies//
+// ============ //
+
+resource resourceGroup 'Microsoft.Resources/resourceGroups@2024-03-01' = {
+  name: resourceGroupName
+  location: resourceLocation
+}
+
+module nestedDependencies './dependencies.bicep' = {
+  scope: resourceGroup
+  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies'
+  params: {
+    actionGroupName: 'dep-${serviceShort}-${namePrefix}-action-group'
+  }
+}
```

```suggestion
// ============ //
// Dependencies //
// ============ //

resource resourceGroup 'Microsoft.Resources/resourceGroups@2024-03-01' = {
  name: resourceGroupName
  location: resourceLocation
}

module nestedDependencies 'dependencies.bicep' = {
  scope: resourceGroup
  name: '${uniqueString(deployment().name, resourceLocation)}-nestedDependencies'
  params: {
    actionGroupName: 'dep-${serviceShort}-${namePrefix}-action-group'
  }
}
```


Last one, and probably a matter of search and replace gone rogue 😄

---

Note: This does does not require a regeneration of any docs